### PR TITLE
Feature decks cards

### PR DESCRIPTION
**Pull request by JWKyle**
Added MVC for decks and cards.  Added basic html to ERBs.

Seeding will be next.

@genoth 
@jalletto

### CODE REVIEW

File: app/models/deck.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Deck < ActiveRecord::Base
+  validates :name, presence: true
+
+  has_many :cards
+  belongs_to :round
```
Comment (genoth) [inline app/models/deck.rb]: Deck 
has_many :rounds

File: app/views/decks/index.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<h1>Decks Index is here</h1>
+
+  <ul>
+    <% @decks.each do |deck| %>
+      <li><a href="/"><%= deck.name %></a></li>
```
Comment (genoth) [inline app/views/decks/index.erb:5]: Hi! Will this href eventually link to the '/decks/:id' page?

**Review by genoth — CHANGES_REQUESTED** [review]
Hi! This looks great. We have 2 change requests, in the comments. :-)

File: app/views/decks/index.erb (hunk 2 above)
Comment (JWKyle) [reply app/views/decks/index.erb:5]: This is a placeholder! We'll keep this in mind once we get to rounds.

**Review by genoth — APPROVED** [review]

**Pull request merged**